lib/term.rb

```diff
@@ -0,0 +1,17 @@
+require 'nokogiri'
+require_relative 'table'
+
+class Term < Page
+
+  field :members do
+    tables.map do |table|
+      Table.new(table).to_a
```

Comment:
There's a slight semantic confusion with naming here, where you have a `Table` class, but `table` is referring to something that isn't a `Table`. I'd suggest calling this inner `table`, something more like `node` to make it clearer that it's simply a nokogiri Node, not an actual Table.